docs/frontend_utilisation_guide.md

```diff
@@ -24,48 +24,91 @@ The Pixel Streaming Frontend UI contains all the functionality for UI components
 
 ---
 
-## Getting Started
-### Installing the Scalable Pixel Streaming Frontend   
+## Scalable Pixel Streaming Frontend Pieces
```

Comment:
Perhaps change this from
> Scalable Pixel Streaming Frontend Pieces

to

> Scalable Pixel Streaming Frontend packages

thoughts?

Reply:
sounds like a good idea. resolved in 10ff7e05bce6278cce6dc9d9b79203fa451dd870